docs/snapshotting/snapshot-support.md

```diff
@@ -618,46 +618,20 @@ The test restores the snapshot and ensures that all the devices set-up in the
 configuration file (network devices, disk, vsock, balloon and MMDS) are
 operational post-load.
 
-The tables below reflect the snapshot compatibility observed on Intel and AMD.
-On ARM, snapshot restore between kernel versions is not possible due to
-registers incompatibility.
-
-### Intel
-
-<table>
-  <tr>
-    <th></th>
-    <th>Snapshot taken on host 4.14</th>
-    <th>Snapshot taken on host 5.10</th>
-  </tr>
-  <tr>
-    <th>Load snapshot on host 4.14</th>
-    <td style="background-color:mediumseagreen">successful</td>
-    <td style="background-color:darkred">unsuccessful due to unresponsive net devices</td>
-  </tr>
-  <tr>
-    <th>Load snapshot on host 5.10</th>
-    <td style="background-color:mediumseagreen">successful</td>
-    <td style="background-color:mediumseagreen">successful</td>
-  </tr>
-</table>
-
-### AMD
-
-<table>
-  <tr>
-    <th></th>
-    <th>Snapshot taken on host 4.14</th>
-    <th>Snapshot taken on host 5.10</th>
-  </tr>
-  <tr>
-    <th>Load snapshot on host 4.14</th>
-    <td style="background-color:mediumseagreen">successful</td>
-    <td style="background-color:mediumseagreen">unsuccessful due to mismatch in MSRs</td>
-  </tr>
-  <tr>
-    <th>Load snapshot on host 5.10</th>
-    <td style="background-color:mediumseagreen">successful</td>
-    <td style="background-color:mediumseagreen">successful</td>
-  </tr>
-</table>
+The tables below reflect the snapshot compatibility observed on the AWS
+instances we support.
+
+**all** means all currently supported Intel/AMD/ARM metal instances (m6g, m7g,
```

Comment:
still a snapshot resume is valid ONLY when a snapshot is take in an architecture and resumed on the SAME cpu architecture. I am not sure this **all** may blury this line which may be worth to be remarked.

Reply:
thanks for the change.